mantis-control-plane/mantis-control-plane-client/src/main/java/io/mantisrx/server/master/client/HighAvailabilityServicesUtil.java

```diff
@@ -62,7 +62,7 @@ public static HighAvailabilityServices createHAServices(CoreConfiguration config
                   "127.0.0.1",
                   apiPort,
                   apiPort + 2,
```

Comment:
remove the +2 from this line too. Mantis master is actually NOT configured to listen on different ports. All paths are served out of 8100.